README.md

```diff
@@ -39,24 +55,148 @@ and obtains its address) will also be written into the config file. Arguments sh
 
 Note, you can use environment variables to pass sensitive data which will not lead to it being written into the config file.
 
+## Security
 
-## Usage example
-Go to `environment` folder.
+Bridge is as long as majority (1/2) of Etherem mining power is honest and supermajority (2/3) of NEAR stake is honest.
+There are no additional security requirements, except that Ethereum should be able to accept 1 transaction within 4 hour period even in the worst congestion scenario.
 
-### Launching blockchains locally
+## Gas costs
 
-First start the services that will emulate locally the NEAR and the Ethereum blockchains:
-## Configs and flags
+NEAR fees are negligible, both for bridge maintenance and for token transfer.
+Ethereum fees are the following:
+* To transfer ERC20 token from ETH to NEAR: Approx 43,989 gas to set allowance and approx 37,407 gas to lock it;
+* To transfer ERC20 token back from NEAR to ETH: Approx 240,531 gas to unlock the token;
+* To submit a NEAR block header: approx 697,140 gas;
+* To challenge a NEAR block header: approx 700k gas.
 
-There are three layers of configuration, overriding each other with highest priorities first:
+As of 2020-07-14 (gas price is 40 gwei) the cost of running bridge on NEAR mainnnet and Ethereum mainnet is approx 42 USD/day. The cost of ETH->NEAR transfer of ERC20 token is 1 USD. The cost of NEAR->ETH transfer of ERC20 token is 2 USD.
 
-1. Environment variables
-2. Flags passed throught the command line
-3. Configuration files in ~/.rainbow/config.json
+## Using existing Testnet bridge
+
+We are currently running bridge between NEAR Testnet and Ethereum Ropsten. The latency on it sped up to allow rapid experimentation.
+Specifically, ETH->NEAR direction is 10 blocks which corresponds to 2-3 minutes latency, and NEAR->ETH direction is ~80 seconds.
+The version of the bridge that will connect NEAR Mainnet to Ethereum Mainnet will use 25 blocks for ETH->NEAR direction and 4 hours for NEAR->ETH direction.
+
+To try cross-blockchain transfer of an existing ERC20 token you would need:
+* Account on Ropsten that owns certain amount of some ERC20 token, in this example we will be using `0x88f975D5A1153Ea92AF66e7c4292576a329c04B6`.
+Ask max@near.org to transfer you some (if you want to transfer your own token you would have to setup your own locker and minting contracts);
```

Comment:
Can you transfer me some :) 0x6858145A1753613C93c2d3e2843687AFC29d0cD6

Reply:
Done! https://ropsten.etherscan.io/tx/0x054974431ab92b8185c29d04027d82328f346590bdf4896d75a1197305babbb6